avidemux_core/ADM_coreUtils/src/prefs2.conf

```diff
@@ -97,7 +97,7 @@ uint32_t:playback,                     0,      0,      4
 Default{
 uint32_t:postproc_type,                0,      0,      7
 uint32_t:postproc_value,               0,      0,      5
-uint32_t:downmixing,                   2,      0,      3
+uint32_t:downmixing,                   3,      0,      4
```

Comment:
Just FYI, I am going to change the default to Stereo.

Anyway, inserting the headphones option means that all existing Avidemux installations which were using the default Dolby Pro Logic II will silently switch to the older Dolby mode (not that anyone is going to notice that ;-)).